library/tests/src/com/google/maps/android/data/kml/KmlPolygonTest.java

```diff
@@ -5,6 +5,7 @@
 import junit.framework.TestCase;
 
 import java.util.ArrayList;
+import java.util.List;
 
 public class KmlPolygonTest extends TestCase {
```

Comment:
Try rebasing this on master branch - there were some changes to the KML package name and method return types in the last release.  That should fix it.